### ISSUE TITLE
ol-cesium 2.1.0 does not work in IE11

### ISSUE THREAD

**Issue by AnthonyChen7** [openlayers/ol-cesium#649]
Hi,

Currently I'm working on an Angular6 application and I'm currently using ol-cesium version 2.1.0 as it seems to be not working in version >= 2.2.0.
When I use some code with OL cesium;

example code snippet:
import OLCesium from 'ol-cesium';
const ol3d = new OLCesium({map: this.olMap});
ol3d.setEnabled(options.enable3D);

I get the following errors when I run it on IE11:
![image](https://user-images.githubusercontent.com/10535970/45908526-daa66880-bdb1-11e8-87c3-02aaea1f75b4.png)

![image](https://user-images.githubusercontent.com/10535970/45908545-f1e55600-bdb1-11e8-889c-777f5d13540d.png)

From looking around on stackoverflow for this kind of error:
https://stackoverflow.com/questions/35808557/javascript-ajax-script1003-expected-in-ie-11

From playing around and testing, we can just make the following fixes and it should be able to work on IE11 and Chrome:
assert(condition) -> assert: function(condition)
assertInstanceof(object, type) -> assertInstanceof: function(object, type)
fail() -> fail: function()
assertNumber(number) -> assertNumber: function(number)

**Comment by samuel-girard** [comment]
Hi @AnthonyChen7,  

This issue about "assert" on IE11 has been solved since ol-cesium 2.3.0 (`goog.assert` calls have been replaced by `console.assert` which works on Internet Explorer).  

Have you still succeeded in using ol-cesium with this browser?  
We are using a similar setup (application on Angular 6), but we have errors in IE11, which we solved with a custom build of ol-cesium.  
I would like to know if anyone have successfully used the current build on IE11 in a similar environment?
